SampleApp/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '11.0'
```

Comment:
If I understand well, the Datadome SDK 3.6.1 requires XCode 15 which seems incompatible with iOS 11 [https://github.com/realm/realm-swift/issues/8368#issuecomment-1725807770](https://github.com/realm/realm-swift/issues/8368#issuecomment-1725807770). Can that cause issues here?

Reply:
Not our problem - we just tell our SDK if for iOS >= 11 - si l'IDE di iOS >= 12, le dev devra supporter iOS >= 12